samples/subsys/fs/littlefs/README.rst

```diff
@@ -88,7 +88,7 @@ To build the test:
    :gen-args: -DCONF_FILE=prj_blk.conf
    :compact:
 
-One can also set ``CONFIG_SDMMC_VOLUME_NAME`` to provide the mount point name
+One can also set ``CONFIG_SDMMC_VOLUME_NAME`` or ``CONFIG_MMC_VOLUME_NAME`` (in case of MMC) to provide the mount point name
```

Comment:
Isn't this line too long?

Reply:
as far as I see, we have even a longer line in this file, the line 57
and GitHub checkpatch hasn't shown any errors